packages/block-editor/src/components/block-list/index.js

```diff
@@ -68,7 +68,7 @@ class BlockList extends Component {
 
 		this.onSelectionStart = this.onSelectionStart.bind( this );
```

Comment:
I'd love if we can somehow isolate/decouple the multi-selection code form the BlockList rendering code.

Reply:
Sounds good. I feel like the PR converting to hooks is a good opportunity to do this.